### ISSUE TITLE
Symbols and suite setup pages

### ISSUE THREAD

**Issue by legatoloco** [unclebob/fitnesse#871]
It seems that setting a symbol in a SetUp page
| $x= | functionCall |
works normally and $x is available and useable from the main test.
This isn't the case for symbols defined the same way but in a SuiteSetUp

Is there any reason why this doesn't work?
Isn't the SuiteSetUp used exactly like the SetUp but runs once per suite? 

How can I initialize a symbol that needs to be done only once per suite? Variables aren't good enough since they can't be dynamically set (as the output of a function)
A SetUp is also not ok as this will run and re-initialize the variable for every test in the suite.

**Comment by ggramlich** [comment]
Here is some background, why it does not work, as you expected.
https://groups.yahoo.com/neo/groups/fitnesse/conversations/topics/17898
TL;DR The SuiteSetUp is run as a different page.

You can work around it, when using some Singleton / Statically stored Map, which keeps state from one page to another and set the values on the SuiteSetUp and read them into symbols again in the SetUp or Test page.

**Comment by legatoloco** [comment]
I actually took your suggestion and implemented my own variables (|setValue; | name | value | and |getValue; | name| ) which worked fine, thanks.

On the other hand, after some more tests, i'm under the impression that the symbols defined in a SuiteSetUp are available in the following pages, they just don't appear visually in the result. But the symbol's value is present. 
My test does an http call and i'm passing a symbol initialized in the SuiteSetUp as a parameter. The called url is receiving the parameter correctly.

**Comment by amolenaar** [comment]
Your assumption is correct: on the FitNesse side, a new context is created per page, but on the SUT side, there is no such thing, hence all variables are available. Both sides are doing their own bookkeeping :(.

**Comment by boredland** [comment]
I know this probably is a dead thread, but: the problem persists. In my case I'd like to perform some variable assignments one time per setup, but that doesn't work very well...
